go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/showwin/speedtest-go
 
-go 1.19
+go 1.24.2
```

Comment:
I think upgrading go should be considered carefully, as it may cause many downstream compatibility issues.

Reply:
unless we need new features or fix someting, we should keep the go version in go.mod as low as possible to maintain maximum compatibility, right?